rust/src/utils.rs

```diff
@@ -194,13 +199,22 @@ pub struct BigNum(u64);
 
 to_from_bytes!(BigNum);
 
+impl std::str::FromStr for BigNum {
+    type Err = JsError;
+    fn from_str(string: &str) -> Result<BigNum, JsError> {
+        string.parse::<u64>()
+            .map_err(|e| JsError::from_str(&format! {"{:?}", e}))
+            .map(BigNum)
+    }
+}
+
 #[wasm_bindgen]
 impl BigNum {
     // Create a BigNum from a standard rust string representation
+    #[allow(clippy::should_implement_trait)]
     pub fn from_str(string: &str) -> Result<BigNum, JsError> {
-        string.parse::<u64>()
-            .map_err(|e| JsError::from_str(&format! {"{:?}", e}))
-            .map(BigNum)
+        // have to redefine so it's visible in WASM
```

Comment:
Won't we be getting rid of this struct in general and just be using `u64` when we drop flow support?

Reply:
Yes, but I just wanted to silence the clippy warning anyway